cms/server/admin/templates/error.html

```diff
@@ -2,7 +2,7 @@
 {% block core %}
 
 <div class="core_title">
-    <h1>{{ "Error %d" % status_code }}</h1>
+    <h1>{{ "Error %d"|format(status_code) }}</h1>
```

Comment:
IIRC in the other commit we decided to use %s?

Reply:
In the other commit we use `{% trans %}`, which is equivalent to `Error %(status_code)s`:
https://github.com/cms-dev/cms/blob/89548be300164b95c79554aa2339163aec460956/cms/server/contest/templates/error.html#L3

Though AWS doesn't use `{% trans %}`, or any localization-related stuff for that matter, so there really is no difference. If you want I can change it.

Reply:
I'd prefer we keep one, so maybe next time we won't go over everything again to make it uniform.